ReactWindows/ReactNative.Shared/UIManager/LayoutShadowNode.cs

```diff
@@ -460,7 +461,7 @@ private static YogaValue ToYogaValue(JValue value)
 
                 if (s.EndsWith("%"))
                 {
-                    return YogaValue.Percent(float.Parse(s.Substring(0, s.Length - 1)));
+                    return YogaValue.Percent(float.Parse(s.Substring(0, s.Length - 1), CultureInfo.InvariantCulture));
```

Comment:
Adding the CultureInfo.InvariantCulture qualifier will ensure that the string being parsed will not be subjected to the culture based number style rules of the locale set on the executing thread.

Reply:
This is the line of code that was creating the FormatException when the percentage style was being used in a view style.